src/main/java/org/fineract/messagegateway/sms/providers/impl/wirepick/smsgateway/Testing.java

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Is this file used for something?

Reply:
Yes it is for the constants provided by the sms provider

Reply:
this is for testing i have not used it but did not want to remove it if we need to do tests in future

Reply:
Then please add a comment in this file that it can be used in future for Tests.

Reply:
But a new one can be made easily as this is an empty class.

Reply:
am removing it thanks for the advise